compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.td

```diff
@@ -781,4 +781,26 @@ def AMDAIE_CircularDmaCpyNdOp: AMDAIE_DmaCpyNdBaseOp<"circular_dma_cpy_nd", [Pur
   let hasVerifier = 0;
 }
 
+def AMDAIE_ReferenceToOp: AMDAIE_Op<"reference_to"> {
+  let summary = "The operation that creates reference to a memref allocation";
+  let description = [{
+    This operation creates a reference to a global memref allocation. It has the same type
+    as the allocation it refers to, but has a local scope that is not hoistable across the AIE tiles.
```

Comment:
Last week we said `is not hoistable across loops ('scf.forall/scf.for')`? Not sure what 'hoistable across the AIE tiles' means?